.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: java
 
+os:
+  - linux
+  - osx
+
 jdk:
+  - oraclejdk11
   - openjdk8
   - openjdk11
 
+matrix:
```

Comment:
If I understand the [documentation](https://docs.travis-ci.com/user/customizing-the-build/#build-matrix) correctly, this should be `jobs:` instead of `matrix:`.

Reply:
Currently that's my best guess why a build (that should be excluded anyway?) fails.

Reply:
Nevermind, I'm just being dumb.